dataland-pgadmin/Dockerfile

```diff
@@ -9,7 +9,9 @@ COPY ./dataland-pgadmin/servers.json /pgadmin4/servers.json
 RUN touch /var/lib/pgadmin/pgpass && chown 5050:5050 /var/lib/pgadmin/pgpass && chmod 0600 /var/lib/pgadmin/pgpass && chmod a+rx /custom_entrypoint.sh
 
 # Install envsubst
-RUN apk add --no-cache gettext
+RUN apk add --no-cache gettext curl
```

Comment:
in `dpage/pgadmin4:6.15` gibt es bereits wget - das können wir statt curl benutzen.

Reply:
Daran habe ich garnicht gedacht. Guter Hinweis. Habe neben dieser Änderung noch dafür gesorgt, dass die HTTP-Response immer im Health-Check-Log steht (mit tee) und mit dem || false erzwinge ich exit code 0 oder 1 (wie vom docker-healthcheck spec verlangt)